### ISSUE TITLE
SwatchColorPicker: Right to Left Support

### ISSUE THREAD

**Issue by Boris-Em** [OfficeDev/office-ui-fabric-react#2851]
### Bug Report

- __Package version(s)__: 5.0
- __Browser and OS versions__: FF, Chrome, Edge, Safari

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? Yes

Requested priority: Low

#### Describe the issue:

The colors in the SwatchColorPicker are not reversed in RTL.

#### Actual behavior:

![screen shot 2017-09-14 at 2 20 56 pm](https://user-images.githubusercontent.com/4613643/30456396-484ed108-9958-11e7-808f-7b52da33d3e1.png)

#### Expected behavior:

The colors should be reversed in RTL.

**Comment by jspurlin** [comment]
This renders perfectly in RTL, the colors are reversed as expected, closing
